tests/unit/snow/test_databricks.py

```diff
@@ -2980,29 +2980,29 @@ def test_to_number(dialect_context):
     validate_source_transpile, _ = dialect_context
     # Test case to validate `TO_DECIMAL` parsing with format
     validate_source_transpile(
-        """SELECT CAST(TO_NUMBER('$345', '$999.00') AS DECIMAL(38, 0)) AS col1""",
+        """SELECT TO_NUMBER('$345', '$999.00') AS col1""",
```

Comment:
practical point: you don't remove existing unit tests, you keep adding new ones.